reign.py

```diff
@@ -30,6 +31,10 @@ def __init__(self, pandas_obj, should_display_map=False):
         self.__msg_cache_handler = None
 
         self.battle_round = 1
+
+        self.threshold = threshold
```

Comment:
Shall we use a better name? What threshold? Shall we call it `empire_size_low_threshold`?

Reply:
yes, sounds good to me!